legislators-current.yaml

```diff
@@ -27274,7 +27274,7 @@
     address: 322 Hart Senate Office Building Washington DC 20510
     phone: 202-224-6542
     fax: 202-228-3027
-    contact_form: http://www.schumer.senate.gov/Contact/contact_chuck.cfm
+    contact_form: http://www.schumer.senate.gov/Contact/email_chuck
```

Comment:
Almost - http://www.schumer.senate.gov/Contact/email_chuck takes me to a blank page, but http://www.schumer.senate.gov/Contact/email-chuck works (dash instead of underscore).